libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -861,7 +861,11 @@ export function generateExpressionFromKey(
   let rootMethod = method;
   if (!isInsideArray && segments[0]?.value?.toString()?.toLowerCase() === OutputSource.Body) {
     segments.shift();
-    rootMethod = actionName ? `${OutputSource.Body}(${convertToStringLiteral(actionName)})` : constants.TRIGGER_BODY_OUTPUT;
+    rootMethod = actionName
+      ? `${OutputSource.Body}(${convertToStringLiteral(actionName)})`
+      : method === constants.TRIGGER_OUTPUTS_OUTPUT
+      ? method
+      : constants.TRIGGER_BODY_OUTPUT;
```

Comment:
We realized this part was causing issues on our end because it was converting things of the format "triggerOutputs()[body/value]" to "triggerBody()[body/value]", making it have double "body" since the result below doesn't remove "body" from the segments. If the method is triggerOutputs, we would like to keep it as trigger outputs. We would like someone from LAUX to confirm this doesn't break spliton triggers or anything else on LogicApps side.

Reply:
Method comes from Redux state which has triggerOutputs in it, but gets converted to triggerBody which is wrong:
![image](https://user-images.githubusercontent.com/8736762/233747013-f7b768ec-1876-40bd-9c98-173b3fe523a7.png)

Reply:
After making this change, we are able to see the option correctly as "@triggerOutputs()?['body/value']"

Reply:
segments[0]?.value?.toString()?.toLowerCase() is validating if first segment is body, so it first segment is body its method is converting into triggerBody(), in your case the first segment shouldn't be coming as body as your segment should be body/value.
Can you point me which caller is calling this generateExpreesionfromKey method?

Reply:
I see the caller now, in your case we should just send aliasing attribute to skip this check, because if we do this for all LA might break;

Reply:
Ahh, just realized it is the OpenApi causing duplicate/repetitive paths for the parameters. Eric (on our team) solved a few of these in other areas. We'll do something similar to that and have it behind the parameterAlias flag. Thank you!

For more context (from Eric):
Each section of the key has the previous one prefixed onto it. It can get really cumbersome:
body.body/value.body/value/foo.body/value/foo/bar
which is really just body -> value -> foo -> bar

Some sample PRs from before:
https://github.com/Azure/LogicAppsUX/pull/1756
https://github.com/Azure/LogicAppsUX/pull/1762
https://github.com/Azure/LogicAppsUX/pull/1725